package.json

```diff
@@ -186,7 +196,7 @@
 				"program": "./out/src/debug/debug_entry.js",
 				"runtime": "node",
 				"variables": {
-					"sdkPath": "dart.getSdkPath"
+					"debugSettings": "dart.getDebugSettings"
```

Comment:
Yeah; the error you get otherwise is really dopey. It says that `sdkPath` is unknown, but since the user never entered that it's not really clear what they should do! It's good this works, but shame about the formatting! Maybe they'll have an idea how to fix it, I'll wait for a response on that Issue before I merge this.